apis/go/go.mod

```diff
@@ -3,16 +3,15 @@ module github.com/seldonio/seldon-core/apis/go/v2
 go 1.18
 
 require (
-	google.golang.org/grpc v1.49.0
+	google.golang.org/grpc v1.56.3
```

Comment:
Latest grpc version is 1.61.0 (released 5 days ago) vs 1.56.3 (released 10 Oct 2023). Looking at https://github.com/grpc/grpc-go/releases it seems there have been some security updates & bugfixes in the meantime. I assume 56.3 is what dependabot proposed? 

We could just choose what dependabot offers to keep things simple

Reply:
yes this is what dependabot **security** workflow suggested. I still have to go over the dependabot **version** upgrade PRs. I will keep it as is for now as I havent changed it manually.